src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, createStore } from 'redux';
+import bookReducer from './books/books';
+import categoryReducer from './categories/categories';
+
+const rootReducer = combineReducers({
+  bookReducer,
+  categoryReducer,
+});
+
+const store = createStore(rootReducer);
+
+export default store;
```

Comment:
- [OPTIONAL] 🤔 What If you Import `applyMiddleware`  to use some handy Redux middleware, like a redux logger.  it will help you a lot in the development process. 👍🏻 To install it, you need to run: 

  ```  npm i --save redux-logger  ```

Reply:
It sounds very interesting, I'm gonna read about it today and see if I can implement it on the next PR.
Thank you!